Pipfile

```diff
@@ -13,6 +13,9 @@ flake8 = "*"
 mypy = "*"
 build = "*"
 isort = "*"
+pytest-cov = "*"
+tox = "<4.0"
+tox-wheel = "*"
 
 [requires]
 python_version = "3.7"
```

Comment:
@harelwa We should remove this to support multiple Python versions.

Reply:
thanks @hirosassa 

indeed

i did not manage to play with pipenv and get a clear “howto get started with development” with any python version without having pipenv change the git state… hence I did not change this yet. i have a bit more digging to do

i’ll be able to cont. work on this PR next wednesday :\  If you’d like to pickup from where I left and complete the PR — by all means. o.w we’ll wait till next week.